dashboard/src/components/Tooltip.tsx

```diff
@@ -0,0 +1,50 @@
+import type React from "react";
+import { useState } from "react";
+import { usePopper } from "react-popper";
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { faInfoCircle } from '@fortawesome/free-solid-svg-icons';
+import clsx from 'clsx';
+
+
+const Tooltip = ({
```

Comment:
might be best to pick a more specific name/make this component more general since we hard coded an "info" icon. Maybe something like "InfoTooltip"?